installer/manifests/keptn/charts/control-plane/Chart.yaml

```diff
@@ -24,7 +24,7 @@ appVersion: latest
 
 dependencies:
   - name: mongodb
-    version: 10.26.4
+    version: 10.31.5
```

Comment:
The new version has some incompatibilities while upgrading. This is not ready to be merged!
See: https://github.com/bitnami/charts/issues/8693